docs/apache-airflow-providers-amazon/operators/lambda.rst

```diff
@@ -0,0 +1,54 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+.. _howto/operator:AwsLambdaInvokeFunctionOperator:
+
+AWS Lambda Invoke Function Operator
```

Comment:
Nitpick. I'm working on standardizing these doc files a bit, if you don't mind.

```suggestion
AWS Lambda Operators
```

Reply:
Not at all. If you have a draft of the docs standardisation (I can view), happy to add anything else I can find.